packages/nextjs/package.json

```
@@ -50,6 +50,7 @@
     "jest": "*",
     "node-fetch-native": "^0.1.8",
     "ts-jest": "*",
+    "tsup": "^6.6.3",
```

Shall we use the tsup version from the root package.json?
```suggestion
    "tsup": "*",
```